packages/gafl-webapp-service/src/processors/renewals-write-cache.js

```diff
@@ -33,14 +33,16 @@ export const setUpCacheFromAuthenticationResult = async (request, authentication
   permission.licenceStartTime = renewedHasExpired ? 0 : endDateMoment.hours()
   permission.renewedEndDate = endDateMoment.toISOString()
   permission.renewedHasExpired = renewedHasExpired
-  permission.licensee = Object.assign(
-    (({ country, preferredMethodOfConfirmation, preferredMethodOfNewsletter, preferredMethodOfReminder, ...l }) => l)(
-      authenticationResult.permission.licensee
-    ),
-    {
-      countryCode: authenticationResult.permission.licensee.country.description
-    }
-  )
+  const copyLicenseeWithoutContactAndCountry = ({
+    country,
+    _preferredMethodOfConfirmation,
+    _preferredMethodOfNewsletter,
+    _preferredMethodOfReminder,
```

Comment:
May need Phil's input on this, will it work with the _ prefixed?

Reply:
It would be worth adding a test to say that the 4 are omitted from `permission.licensee`